### PR TITLE
move channel selection to top of publish form

### PR DESCRIPTION
**Pull request by seanyesmunt**
Maybe a little less confusing this way

### CODE REVIEW

File: static/app-strings.json [hunk 1]
```diff
@@ -1567,5 +1567,6 @@
   "Delete or edit past content to spend": "Delete or edit past content to spend",
   "Delete supports to spend": "Delete supports to spend",
   "%lbc_amount% boosting content": "%lbc_amount% boosting content",
+  "Deposit cannot be higher than your available balance: %balance%": "Deposit cannot be higher than your available balance: %balance%",
```
Comment (DispatchCommit) [inline static/app-strings.json:1570]: Why is this included in the PR?

**Review by DispatchCommit — APPROVED** [review]
LGTM, it's a nice improvement.

File: static/app-strings.json (hunk 1 above)
Comment (seanyesmunt) [reply static/app-strings.json:1570]: Any time the desktop app is run locally, it populates this file with language strings if it's missing any.

This string wasn't added to the file so it was added automatically. (It should have been added when that string was originally added to the UI but I forgot)
